_about/notes/0.5.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Istio 0.5
+
+order: 96
+
+layout: about
+type: markdown
+---
+{% include home.html %}
+
+In addition to the usual pile of bug fixes and perf improvements, this release includes the new or 
+updated features detailed below.
+
+## Networking
+
+- **Alpha2 Traffic Rules**. Add support for an updated model of routing and destination rules. TODO: Doc link?
+
+- **Control Plane Deployments**. `istioctl` now has a deployment command to produce istio control plane deployments. TODO: Doc link?
+
+- **Automatic Proxy Injection**. Use Kubernetes 1.9's mutating webhook functionality to provide automatic
+pod-level proxy injection. Automatic injection doesn't work requires Kubernetes 1.9 or beyond and
```

Comment:
punctuation missing between "Automatic injection doesn't work" and "requires Kubernetes 1.9" ?

Reply:
Doc link: https://istio.io/docs/setup/kubernetes/sidecar-injection.html#automatic-sidecar-injection

Reply:
Done